QamarDeen/QamarDeen/Model/Generated/Day+CoreDataProperties.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Day+CoreDataProperties.swift
+//  QamarDeen
+//
+//  Created by Mazyad Alabduljaleel on 12/25/16.
+//  Copyright © 2016 Batoul Apps. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension Day {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Day> {
+        return NSFetchRequest<Day>(entityName: "Day");
+    }
+
+    @NSManaged public var hijriDate: NSDate?
```

Comment:
Well a string would make it difficult to query for specific date ranges. 

I now remember that yes, we converted the date to midnight UTC and then stored it. Dates are tricky things, so lets just make sure to think carefully when doing the manipulations.